### PR TITLE
Update `get_data` to us `get_some_data` sub  functions.

### PR DESCRIPTION
**Pull request by UpstreamData**
Updated from PR #26, as I messed something up with git that broke all changes.

### CODE REVIEW

File: pyasic/API/btminer.py [hunk 1]
```diff
@@ -406,7 +450,7 @@ async def set_led(
         </details>
         """
         if auto:
-            return await self.send_privileged_command("set_led", param="auto")
```
Comment (crc32) [inline pyasic/API/btminer.py:409]: Check that this is working with modern Whatsminers

File: pyasic/miners/antminer/bmminer/X19/X19.py [hunk 2]
```diff
@@ -25,131 +25,74 @@
...
-                await client.post(url, data=conf, auth=auth)  # noqa - ignore conf being a str
-        except httpx.ReadTimeout:
-            pass
         for i in range(7):
             data = await self.get_config()
             if data.as_x19() == conf:
                 break
             await asyncio.sleep(1)
 
-    async def get_hostname(self) -> Union[str, None]:
```
Comment (crc32) [inline pyasic/miners/antminer/bmminer/X19/X19.py:74]: v

Comment (crc32) [inline pyasic/miners/antminer/bmminer/X19/X19.py:74]: Where did get_hostname go? Realizing that I'm looking through this much later and that doing it in person would have been more efficient

Comment (UpstreamData) [reply pyasic/miners/antminer/bmminer/X19/X19.py:74]: I believe it got moved up a level maybe?  I can find an S19 to test on and call `get_hostname` on it tomorrow, unless you want to do it tonight?

Comment (UpstreamData) [reply pyasic/miners/antminer/bmminer/X19/X19.py:74]: https://github.com/UpstreamData/pyasic/blob/dev-fixed/pyasic/miners/antminer/bmminer/X19/X19.py

Line 108.

**Pull request merged**